Workbooks/Images/Logos/sophos_logo.svg

```diff
@@ -0,0 +1 @@
41.939 8.031 2.874 10.756 2.577 3.719 7.585 5.802 13.929 5.802 4.461 0 8.229-1.093 10.708-3.075 3.271-2.625 5.108-7.388 5.108-13.188l-.015-.946-.04-.931-.066-.906-.089-.889-.116-.867-.144-.84-.165-.813-.191-.787-.104-.384-.111-.375-.116-.371-.121-.357-.125-.356-.134-.345-.14-.334-.146-.332-.147-.316-.155-.312-.161-.303-.166-.291-.174-.285-.175-.277-.183-.264-.188-.254C60.523 1.836 55.811 0 49.912 0zm.048 5.599c5.899 0 8.526 3.473 8.526 11.204 0 7.634-2.677 11.153-8.526 11.153-5.947 0-8.625-3.521-8.625-11.252l.004-.326.004-.322.005-.316.013-.312.014-.305.014-.303.018-.295.022-.291.026-.287.027-.277.031-.271.035-.268.036-.264.04-.26.045-.25.045-.25.05-.24.054-.232.054-.232.059-.224.062-.22.062-.215.066-.205.072-.201.076-.196.076-.192.081-.183.085-.183.084-.174.089-.166.094-.162.094-.155c1.339-2.134 3.867-3.328 7.088-3.328zM69.99.25v33.062h7.034V21.758H88.33c3.767 0 5.546-.492 7.48-2.128 2.28-1.932 3.669-5.154 3.669-8.625 0-2.924-1.04-5.652-2.973-7.734C94.326.842 92.291.25 86.492.25zm7.034 5.899h9.715c1.14 0 2.033.097 2.578.296 1.538.545 2.431 2.131 2.431 4.311 0 1.835-.547 3.422-1.438 4.166l-.071.062-.072.061-.076.059-.075.059-.081.053-.08.05-.085.049-.089.046-.09.043-.094.045-.094.037-.098.039-.104.036-.103.029-.107.032-.112.031-.115.025-.116.021-.121.022-.125.022-.129.018-.13.018-.139.014-.139.018-.143.01-.152.009-.152.01-.156.009-.162.003-.166.005h-.17l-.175.004h-9.567V6.149zm45.503 13.53h-12.738v13.629h-7.039V.25h7.039v13.531h12.738V.25h7.039v33.058h-7.039zM150.088 0c-4.658 0-8.723 1.141-11.301 3.223-3.271 2.578-5.01 7.387-5.01 13.732 0 4.41.943 8.031 2.879 10.756 2.574 3.719 7.582 5.802 13.928 5.802 4.463 0 8.227-1.093 10.703-3.075 3.271-2.625 5.107-7.388 5.107-13.188l-.014-.946-.039-.931-.066-.906-.09-.889-.117-.867-.143-.84-.166-.813-.188-.787-.107-.384-.107-.375-.115-.371-.125-.357-.125-.356-.135-.345-.139-.334-.143-.332-.152-.316-.156-.312-.16-.303-.166-.291-.174-.285-.174-.277-.186-.264-.188-.254C160.693 1.836 155.988 0 150.088 0zm.051 5.599c5.896 0 8.525 3.473 8.525 11.204 0 7.634-2.678 11.153-8.525 11.153-5.947 0-8.625-3.521-8.625-11.252v-.326l.004-.322.01-.316.008-.312.014-.305.02-.303.018-.295.021-.291.021-.287.027-.277.035-.271.031-.268.035-.264.041-.26.043-.25.047-.25.049-.24.053-.232.055-.232.059-.224.062-.22.062-.215.07-.205.07-.201.072-.196.076-.192.08-.183.086-.183.084-.174.09-.166.094-.162.1-.155c1.336-2.134 3.865-3.328 7.088-3.328zm20.42 21.813h15.068c2.625 0 4.062-.248 4.906-.793 1.039-.646 1.635-1.733 1.635-3.022 0-1.537-.793-2.876-2.131-3.521-.695-.396-1.982-.595-3.568-.595h-6.098c-3.965 0-6.543-.694-8.277-2.181-1.982-1.736-3.123-4.412-3.123-7.286 0-4.213 2.328-7.881 5.75-9.071 1.486-.546 3.223-.693 7.088-.693h16.357v5.898h-14.771c-3.123.049-3.371.049-4.41.297-1.389.346-2.232 1.635-2.232 3.369 0 1.489.695 2.678 1.887 3.223.988.447 1.781.545 4.162.545h5.303c4.857 0 7.484.744 9.42 2.727C199.01 17.795 200 20.72 200 23.596c0 3.519-1.635 6.789-4.213 8.376-1.635 1.04-3.717 1.337-8.625 1.337h-16.604v-5.897z" fill="#007aff"/></svg>
```

Comment:
Format the SVG file - take it to VS Code and shift-alt 'F' to format

Reply:
Applied the follow requests, please check if I did this correctly.